chandra_aca/star_probs.py

```diff
@@ -678,11 +678,11 @@ def guide_count(mags, t_ccd, count_9th=False):
     ref_mags0 = (9.0 if count_9th else 9.95) + np.array([0.0, 0.2, 0.3, 0.4])
     ref_mags_t_ccd = [snr_mag_for_t_ccd(t_ccd, ref_mag, ref_t_ccd) for ref_mag in ref_mags0]
 
-    # The 5.85 and 5.95 limits are not temperature dependent, these reflect the
-    # possibility that the star will be brighter than 5.8 mag and the OBC will
+    # The bright limits are not temperature dependent, these reflect the
```

Comment:
I'd rather just revert to the original comments adjusted by 0.6 as everywhere else. In both cases the use of "bright" requires some context and is not 100% obvious without thinking.

Reply:
Sounds good.  Though I suppose I was confused by this to start because 5.85 and 5.95 weren't really limits.  But I'll just update as you say and can revise the comment in the future if we are here again.